qa/rpc-tests/sc_cert_getblocktemplate.py

```diff
@@ -138,25 +149,30 @@ def run_test_with_scversion(self, scversion, ceasable = True):
 
         ### 1: Creating a transaction
         node2_address = self.nodes[2].getnewaddress()
-        mark_logs("\nCall GetBlockTemplate on each node to create a cached (empty) version; check it is empty", self.nodes, DEBUG_MODE)
+        mark_logs("\nCall GetBlockTemplate on each node to create a new cached version; check it is empty", self.nodes, DEBUG_MODE)
         for i in range(0, NUMB_OF_NODES):
             cacheTime[i] = time.time()
             blocktemplate = self.nodes[i].getblocktemplate()
             assert_equal(len(blocktemplate['certificates']), 0)
             assert_equal(len(blocktemplate['transactions']), 0)
```

Comment:
If we really want to save a bit more ms, we could collect `cacheTime[i]` and `blocktemplate[i]` in this loop, and then do the assertions and print in another, separate loop.

However, my proposal is to keep the code as it is and just remove the final assertion on the number of unexpected_too_long_op. 
The reason behind that is that we cannot control how much time `getblocktemplate()` takes to provide a result on every execution environment this test is executed in, so we can expect that we would not be able to check the inclusion of txs and certs may be delayed.
I would just print a message at the end of the execution, warning that  at least one check took too long, in case this happens.

Reply:
Regarding the assertions I agree with you but probably the saving would be very very minimal and not worth the effort.

I have the same feeling the final assertion on `unexpected_too_long_op` should be removed, adding a warning message is a good way to at least keep track of the behaviour in case an analysis is needed. I'll go with your suggestions.

Reply:
Totally agree! Thank you